cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,26 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'TestingOvsianka',
+    password: 'Qwerty@12345'
+  };
 
+  it('should allow to login', () => {
```

Comment:
there is no assertion for that task - "assert your username is shown after login"